main.py

```diff
@@ -130,8 +141,53 @@ def gameLoop():
         while pyOver:
             image = pygame.image.load(python_path)
             game_display.blit(image, (0, 0))
-            message("Game Over! Press C to play Again, Q to Quit", (255, 0, 0), 30)
-            message(lossreason, (255, 0, 0), 55)
+            
+            message("Game Over! Press C to play Again, Q to Quit", (255, 0, 0), -20)
+            message(lossreason, (255, 0, 0), 30)
+            #display score on game over
+            message("Your"+scorestr,(255,0,0),80)
+            if tot > highscore:
+                #message("Highscore!!!",(255,0,0),120)
+                #write new highscore
+                highscorefile = open('highscore.txt','wt')      
+                highscorefile.write(str(tot)+"\n")
+                
+                #name window
+                def namewrite():
+                    highscorefile.write(v.get())
+                    scorewindow.destroy()
+                
+                scorewindow = Tk()
+                scorewindow.geometry('300x100')
+                frame=Frame(scorewindow,width=100,height=100)
+                frame.pack()
+                scorewindow.title("congratulations")
+                
+                
+                Label(frame, text='you\'ve made highscore!!!!').pack(side='top')
+                v = StringVar()
+                v.set("type your name")
+                textbox = Entry(frame, textvariable = v)
+                textbox.pack(side='top')
+                
+                butok=Button(frame,text="ok",fg="black",bg="white", command = namewrite)
```

Comment:
Learn to write proper camelcased variable names. Just building a feature isn't enough. Always remember that there will be many devs working on the same project